apps/stripe/src/app/api/stripe/webhook/stripe-object-handlers/stripe-payment-intent-handler.test.ts

```diff
@@ -158,7 +158,7 @@ describe("StripePaymentIntentHandler", () => {
           const amountToUse = 123_367;
           const amountExpected = 12.3367; // Converted to Saleor float
 
-          event.data.object.amount_received = amountToUse;
+          event.data.object.amount = amountToUse;
```

Comment:
on pairing found out that for this event it must be amount, not received (its before charge/capture yet)